src/pages/Homepage/Homepage.vue

```diff
@@ -30,16 +30,14 @@ export default {
 		};
 	},
 	inject: ['apollo'],
-	apollo: {
-		query: promoQuery,
-		variables() {
-			return {
+	mounted() {
+		this.apollo.query({
```

Comment:
By not prefetching, will the hero-slideshow component start showing the default content first and then show the promo content once the contentful response comes in?

Reply:
See update, but I'm open to suggestions for other ways to do it

Reply:
I think this works really well as a stopgap. Having Contentful cached on a backend server we control would probably be an ideal scenario, but this will work well to stop the bleeding.